### ISSUE TITLE
Encouraging diversity in WebVR development

### ISSUE THREAD

**Issue by billyco** [w3c/webvr-content-workshop#27]
I would like to discuss how we could encourage of more diverse group to get involved with WebVR. I would like to do some kind of initiative in my country and want to learn about other participants' experience in this area.

**Comment by delapuente** [comment]
Fair point. I think we could learn from sharing experiences. I would make this a table in the follow-up sessions. Would you like to host this conversation, @billyco ?

**Comment by ktrogolo** [comment]
I can host/lead this conversation if @billyco does not. I recently founded a Women's XR group and am working with other organizations in efforts to encourage a diverse WebVR talent pipeline.

**Comment by billyco** [comment]
I was going to suggest @ktrogolo to lead the conversation here and I have a few more participants in mind I would like to involve.

**Comment by Malwine** [comment]
Hello everybody, hi @billyco ! 😊 
I would love to join the discussion. With some friends I started the diversity initiative [ClojureBridge Berlin](http://clojurebridge-berlin.org/) and can share some insights about it and diversity in general. Currently I am doing a WebVR [project](https://wit-exhibition.github.io/) to give visibility to women and non-binary people in computer science.
